pkg/parser/digester.go

```diff
@@ -408,6 +456,22 @@ func (d *sqlDigester) reduceInListWithSingleLiteral(currTok *token) {
 	}
 }
 
+func (d *sqlDigester) reduceInRowListWithSingleLiteral(currTok *token) {
```

Comment:
Please add some comments for this function.

Reply:
I updated.